finish/src/main/java/io/openliberty/guides/bulkhead/global/eBank/microservices/rest/Producer.java

```diff
@@ -8,7 +8,7 @@
  * Contributors:
  *     IBM Corporation - initial API and implementation
  *******************************************************************************/
-package global.eBank.microservices.rest;
+package io.openliberty.guides.bulkhead.global.eBank.microservices.rest;
```

Comment:
Same comment as before for the rest directory to be under global.eBank rather than global.eBank.microservices.